x-pack/plugins/apm/public/store/reactReduxRequest/serviceList.tsx

```diff
@@ -5,19 +5,31 @@
  */
 
 import React from 'react';
+import { Request, RRRRender, RRRRenderResponse } from 'react-redux-request';
+import { IApmService } from 'x-pack/plugins/apm/public/components/app/ServiceOverview/ServiceList';
 import { loadServiceList } from '../../services/rest/apm';
-import { Request } from 'react-redux-request';
+import { IReduxState } from '../rootReducer';
+import { IUrlParams } from '../urlParams';
+// @ts-ignore
 import { createInitialDataSelector } from './helpers';
 
 const ID = 'serviceList';
-const INITIAL_DATA = [];
+const INITIAL_DATA: [] = [];
```

Comment:
Hm ... I guess? It's weird to me to type something I'm hard-coding the value of right there, esp since this value is always replaced/never mutated. I was surprised TS couldn't just infer the type?